public/controllers/register-agent/components/steps-three/group-input.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { Fragment, useState } from 'react';
+import {
+  EuiComboBox,
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPopover,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { webDocumentationLink } from '../../../../../common/services/web_documentation';
+import { PLUGIN_VERSION_SHORT } from '../../../../../common/constants';
+
+const popoverAgentGroup = (
+  <span>
+    Learn about{' '}
+    <a
+      href={webDocumentationLink(
+        'user-manual/reference/ossec-conf/client.html#groups',
+        PLUGIN_VERSION_SHORT,
+      )}
+      target='_blank'
+      rel='noopener noreferrer'
+    >
+      Select a group.
+    </a>
+  </span>
+);
+
+const GroupInput = ({ value, options, onChange }) => {
+  const [isPopoverAgentGroup, setIsPopoverAgentGroup] = useState(false);
+
+  const onButtonAgentGroup = () =>
+    setIsPopoverAgentGroup(isPopoverAgentGroup => !isPopoverAgentGroup);
+  const closeAgentGroup = () => setIsPopoverAgentGroup(false);
+  return (
+    <>
+      <EuiFlexGroup>
+        {/* <EuiFlexItem
+          style={{
+            marginTop: '32px',
+            flexDirection: 'row',
+            fontStyle: 'normal',
+            fontWeight: 700,
+            fontSize: '12px',
+            lineHeight: '20px',
+            color: '#343741',
+          }}
+          data-testid='group-input-label'
+        >
+          <p>Select one or more existing groups</p>
+          <EuiIconTip
+            content='Source maps allow browser dev tools to map minified code to the original source code'
+            position='right'
+          />
+        </EuiFlexItem> */}
```

Comment:
**nitpick**: remove this comment

Reply:
Removed in commit [777d15d](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/777d15df38e5e1f1d42b05b49a9cdd8f3ccbf29d)